pkg/keb/test/cluster.go

```diff
@@ -10,23 +10,39 @@ import (
 )
 
 func NewCluster(t *testing.T, runtimeID string, clusterVersion uint64, newConfigVersion bool, clusterType Cluster) *keb.Cluster {
+	runtimeID += uuid.NewString()
```

Comment:
I would ust use the uuid and remove the runtimeId as input

Reply:
That will involve many code refactor, and to allow manually adding something to runtimeID can help during debug (search that prefix in runtimeID)

Reply:
But maybe better change the `runtimeID` to `runtimeIDPrefix` so that not leave the wrong impression that this ID is same as given input. What do you think?